listener.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"log"
+	"net"
+
+	baps3 "github.com/UniversityRadioYork/baps3-go"
+)
+
+type Listener struct {
+	listener    net.Listener
+	connectorCh chan baps3.Message
+	clientComm  chan []byte
```

Comment:
Would these two channels be better names responseCh and requestCh as in baps3.Connector?